flexible_plate_dambreak_2D/dambreak.py

```diff
@@ -399,7 +399,7 @@ def get_w(self):
         return 0.
     def calculate_init(self):
         self.last_F = None
-        self.calculate()
+        #self.calculate()
```

Comment:
@Milad-Rakhsha iirc this call to calculate() occurs too early  in the initialization to have the pressure attached. I don't think it's actually necessary to call it here, so I just commented it out.

Reply:
So I am getting the same segfault in the flexible plate and the cylinder drop examples, but you mentioned that you are already running the cylinder drop example. Would it be possible to share that test case? I think it would help me a lot to compare what you have vs what I have for that example and the would lead me to solve the segfault example in the flexible plate example as well. I believe that the two segfaults are essentially caused by the same issue

Reply:
To be clear, the only reason I changed the sdf calculation to the internal distance calculation was because of the speed. I could run it in either mode.